apps/src/lab2/views/components/TextToSpeech.tsx

```diff
@@ -0,0 +1,39 @@
+import {markdownToTxt} from 'markdown-to-txt';
+import React, {useCallback} from 'react';
+
+import FontAwesomeV6Icon from '@cdo/apps/componentLibrary/fontAwesomeV6Icon/FontAwesomeV6Icon';
+
+import moduleStyles from './TextToSpeech.module.scss';
+
+interface TextToSpeechProps {
+  text: string;
+}
+
+/**
+ * TextToSpeech play button.
+ */
+const TextToSpeech: React.FunctionComponent<TextToSpeechProps> = ({text}) => {
+  const playText = useCallback(() => {
+    const plainText = markdownToTxt(text);
+    const utterance = new SpeechSynthesisUtterance(plainText);
+    speechSynthesis.cancel();
+    speechSynthesis.speak(utterance);
+  }, [text]);
+
+  return (
+    <div
```

Comment:
Can we make this a button?

Reply:
Does it give additional functionality?  I tried that initially, and while it's good that the shadow on hover no longer needs to be overridden, we'd still need to override the background color, the border, and the border when active.

Reply:
Just pulled locally and it appears I'm not actually able to focus (at least visibly) and interact with this via keyboard on Chrome/Safari, do you repro? Works if I switch to `button` element.

Reply:
You should be able to use the [@remove-button-styles](https://github.com/code-dot-org/code-dot-org/blob/6f24c08bf35ef6dc4e6068d5ef353737f3c2ee8b/apps/src/mixins.scss#L3) mixin to remove all the default application button styling.

Also depending on the final styling, we could also use an icon-only DSCO button maybe?

Reply:
Updated to use a `button`.